apps/challenges/aws_utils.py

```diff
@@ -1242,13 +1262,17 @@ def create_eks_cluster_subnets(challenge):
     Arguments:
         instance {<class 'django.db.models.query.QuerySet'>} -- instance of the model calling the post hook
     """
-    from .models import ChallengeEvaluationCluster
-    from .serializers import ChallengeEvaluationClusterSerializer
+    from .models import ChallengeEvaluationCluster, ChallengePhase
+    from .serializers import (
+        ChallengeEvaluationClusterSerializer,
+        ChallengePhaseCreateSerializer,
+    )
     from .utils import get_aws_credentials_for_challenge
 
     for obj in serializers.deserialize("json", challenge):
         challenge_obj = obj.object
     challenge_aws_keys = get_aws_credentials_for_challenge(challenge_obj.pk)
+    environment_suffix = "{}-{}".format(challenge_obj.pk, settings.ENVIRONMENT)
```

Comment:
Let's also add the phase pk so that we know which phase does it correspond to.

Reply:
We can't add this unless we switch to 1 EFS per phase in this PR itself